### DISCUSSION TITLE
Blank Sniffer Window... What am I doing wrong???

### DISCUSSION THREAD

**Discussion by FrankM54** [collin80/SavvyCAN#304] [Q&A]
How do you get data into the "Sniffer" window?

 I load a CAN log (from SavvyCan) and can see, analyze and graph data.
Then I click on RE Tools > Sniffer, a window opens with no CAN bus data and I have no idea how to load data into it.

<img width="1324" alt="image" src="https://user-images.githubusercontent.com/77734337/109433376-ff20b900-79dd-11eb-8033-7451e33dc66a.png">

I have tried this on OSX v199 and Linux (Ubuntu) v202 (1.0.150) with the same results.
What am I missing?

Thanks in advance.

**Comment by collin80** [comment]
The sniffer window is temporal. You have to have traffic flowing for it to show anything. You can simulate this by loading your capture file into the playback window, select "Use Original Timing" and play it. You'd have to have some sort of bus set up but it can be a virtual bus. Then the frames played back on that virtual bus will work to trigger the sniffer window to operate.

**Reply by FrankM54** [reply]
I understand and thanks for the quick reply!
